venues/ICLR.cc/2019/Workshop/DeepGenStruct/python/review-assignment-stage.py

```diff
@@ -0,0 +1,23 @@
+import openreview
+import config
+import argparse
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+conference = config.get_conference(client)
+
+print ("Set global committee members")
+conference.set_authors()
+conference.set_program_chairs(['pc@mail.com'])
+conference.set_area_chairs(['mbok@cs.umass.edu'])
+conference.set_reviewers(['pmandler@cs.umass.edu'])
+
+print ("Create some assignments")
+conference.set_assignment('mbok@cs.umass.edu', 1, is_area_chair = True)
+conference.set_assignment('mandler@cs.umass.edu', 1, is_area_chair = False)
```

Comment:
Is this just checked in as a placeholder?  It seems like code that will either be done in Jupyter notebook or read in assignments from a spreadsheet.  I don't think there are area chairs for this workshop.

Reply:
It is an example, I can remove this code.

Reply:
I removed the area chair